src/main.js

```diff
@@ -507,11 +507,13 @@ class MiniGraphCard extends LitElement {
   }
 
   renderSvg() {
-    const { height } = this.config;
+    const { height, show } = this.config;
+    const grid_lines_type = show.grid_lines_type ? show.grid_lines_type : false;
     return svg`
       <svg width='100%' height=${height !== 0 ? '100%' : 0} viewBox='0 0 500 ${height}'
         @click=${e => e.stopPropagation()}>
         <g>
+          ${grid_lines_type ? this.renderGridLines() : ''}
```

Comment:
Let's place all grid lines into a separate `<g>` element with a defined class:
1. Move this line BEFORE the `<g>` line (like it is done for `renderSvgPoints`).

2. In `renderGridLines` please try doing this (taken from `renderSvgPoints`):
```
    return svg`
      <g class='grid-lines'
        ... show lines
      </g>`;
```
The goal is to place all `<line>` elements into a separate `<g>` element with a defined `grid-lines` class. This will allow users to easier use card-mod; also this will allow developers to debug the appearance of the card in Code Inspector.

If this is hard to implement - please tell.

Reply:
Done